packages/big-design/src/components/Checkbox/Checkbox.tsx

```diff
@@ -3,14 +3,21 @@ import React, { forwardRef, useMemo, Ref } from 'react';
 
 import { useUniqueId } from '../../hooks';
 import { typedMemo, warning } from '../../utils';
+import { FormControlDescription, FormControlDescriptionLinkProps } from '../Form';
 
-import { CheckboxContainer, HiddenCheckbox, StyledCheckbox } from './styled';
+import { CheckboxContainer, CheckboxLabelContainer, HiddenCheckbox, StyledCheckbox } from './styled';
 import { CheckboxLabel } from './Label';
 
 interface Props {
   hiddenLabel?: boolean;
   isIndeterminate?: boolean;
   label: React.ReactChild;
+  description?: CheckboxDescription;
```

Comment:
Since most of the time we won't have links, what do you guys think having this as `string | CheckboxDescription`?

Reply:
I agree